quickstart/src/index.js

```diff
@@ -1,223 +1,150 @@
+const $joinRoomModal = $('#join-room', $modals);
+
+const connectOptions = isMobile ? {
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  bandwidthProfile: {
+    video: {
+      dominantSpeakerPriority: 'high',
+      maxSubscriptionBitrate: 2500000,
+      mode: 'collaboration',
+      renderDimensions: {
+        high: { height: 720, width: 1280 },
+        standard: { height: 90, width: 160 }
+      }
+    }
+  },
+
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  dominantSpeaker: true,
+
+  // Uncomment this line to enable verbose logging.
+  // logLevel: 'debug'
+
+  // Comment this line if you are playing music.
+  maxAudioBitrate: 16000,
+
+  // Comment this line if you are in a Peer-to-Peer Room.
+  preferredVideoCodecs: [{ codec: 'VP8', simulcast: true }],
+
+  // For mobile browsers, capture 360p video @ 24 fps.
+  video: { height: 720, frameRate: 24, width: 1280 }
+} : {
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  bandwidthProfile: {
+    video: {
+      dominantSpeakerPriority: 'high',
+      mode: 'collaboration',
+      renderDimensions: {
+        high: { height: 720, width: 1280 },
+        standard: { height: 90, width: 160 }
+      }
+    }
+  },
 
-// A new RemoteTrack was published to the Room.
-function trackPublished(publication, container) {
-  if (publication.isSubscribed) {
-    attachTrack(publication.track, container);
-  }
-  publication.on('subscribed', function(track) {
-    log('Subscribed to ' + publication.kind + ' track');
-    attachTrack(track, container);
-  });
-  publication.on('unsubscribed', detachTrack);
-}
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  dominantSpeaker: true,
 
-// A RemoteTrack was unpublished from the Room.
-function trackUnpublished(publication) {
-  log(publication.kind + ' track was unpublished.');
-}
+  // Comment this line to disable verbose logging.
+  logLevel: 'debug',
 
-// A new RemoteParticipant joined the Room
-function participantConnected(participant, container) {
-  let selfContainer = document.createElement('div');
-  selfContainer.id = `participantContainer-${participant.identity}`;
+  // Comment this line if you are playing music.
+  maxAudioBitrate: 16000,
 
-  container.appendChild(selfContainer);
-  appendName(participant.identity, selfContainer);
+  // Comment this line if you are in a Peer-to-Peer Room.
+  preferredVideoCodecs: [{ codec: 'VP8', simulcast: true }],
```

Comment:
Why? Would it hurt in some way? Would not simulcast setting just be ignored in case of p2p? An explanation would help.

Reply:
@anna-vasilko Simulcast would still be applied in Peer-to-Peer Rooms but there is no SFU to forward only one of the layers to the remote party. So all 3 layers will be sent, which is a waste of bandwidth.